ndcube/ndcube.py

```diff
@@ -321,43 +321,47 @@ def _generate_pixel_grid(self, edges, wcs):
         return np.meshgrid(*ranges, indexing='ij', sparse=sparse)
 
     @utils.misc.sanitise_wcs
-    def axis_world_coords(self, *axes, edges=False, wcs=None):
+    def axis_world_coords(self, *axes, pixel_corners=False, wcs=None):
         """
         Returns WCS coordinate values of all pixels for all axes.
 
         Parameters
         ----------
-        axes: `int` or `str`, or multiple `int` or `str`
+        axes: `int` or `str`, or multiple `int` or `str`, optional
             Axis number in numpy ordering or unique substring of
             `~ndcube.NDCube.world_axis_physical_types`
             of axes for which real world coordinates are desired.
             axes=None implies all axes will be returned.
 
-        edges: `bool`
-            The edges argument helps in returning `pixel_edges`
-            instead of `pixel_values`. Default value is False,
-            which returns `pixel_values`. True return `pixel_edges`
+        pixel_corners: `bool`, optional
+            If `True` then instead of returning the coordinates of the pixel
+            centers the coordinates of the pixel corners will be returned, this
+            increases the size of the output by 1 as all corners are returned.
```

Comment:
```suggestion
            If `True` then instead of returning the coordinates at the centers of the pixels,
            the coordinates at the pixel corners will be returned. This
            increases the size of the output by 1 in all dimensions as all corners are returned.
```